engines/dfc_provider/spec/requests/enterprises_spec.rb

```diff
@@ -73,6 +73,13 @@
               %r{active_storage/[0-9A-Za-z/=-]*/logo.png},
               "active_storage/url/logo.png",
             )
+            response.body.gsub!(
```

Comment:
The above replacements were chained, it would be good if we choose to follow that convention or not, and update accordingly.
I'm not sure it's intuitive to chain a destructive `!` method, but I like that it reduces repetition. So I'd opt for chaining but don't really mind either way.

```suggestion
            ).gsub!(
```